gateway/server.go

```diff
@@ -1765,6 +1766,10 @@ func (gw *Gateway) start() {
 		go gw.startPubSubLoop()
 	}
 
+	oauthTokensPurger := scheduler.NewScheduler("purge-oauth-tokens", gw.GetConfig().Private.GetOAuthTokensPurgeInterval(),
+		log, gw.purgeLapsedOAuthTokens)
+	go oauthTokensPurger.Start(gw.ctx)
+
 	conf := gw.GetConfig()
```

Comment:
There's a conf already declared here, optimize the above gw.GetConfig (use a temp var for it, have NewScheduler on a single line

Reply:
Ideally I would make it to use across the whole func